apps/web/src/server/applications/create-new-case/applicant/applications-create-applicant.validators.js

```diff
@@ -47,6 +47,5 @@ export const validateApplicationsCreateApplicantPostCode = createValidator(
 			/^([Gg][Ii][Rr] 0[Aa]{2})|((([A-Za-z]\d{1,2})|(([A-Za-z][A-HJ-Ya-hj-y]\d{1,2})|(([A-Za-z]\d[A-Za-z])|([A-Za-z][A-HJ-Ya-hj-y]\d[A-Za-z]?))))\s?\d[A-Za-z]{2})$/
```

Comment:
this could use - https://express-validator.github.io/docs/api/validation-chain/#ispostalcode

There will be International postcodes.

Reply:
@jonathangoulding This is pre-existing code. The change is due to my auto formatting on save. isPostalCode is being used for the validation on the new page.